eng/validation/templates/testing/helix.yml

```diff
@@ -0,0 +1,46 @@
+jobs:
+  - job:
+    displayName: Validate Helix
+    variables: 
+    - name: _BuildConfig
```

Comment:
Is it worth declaring these as variables? It looks like we could just pass these values directly to the template.

Reply:
I left it like this since we might want to add another helix step which also makes use of the same so we don't have to change the value in two (N) places if there's a need for a change. Valid point tho'. For know I'll leave them there

Reply:
This is a template?  Could you move most of these to be parameters in this template and set reasonable templates then pass those values through to the send-to-helix.yml template?  That would look a bit cleaner.

Reply:
Agree with Chris. If we want to reuse this from the base azure pipelines yaml, then the ones that will always be the same should stay this way, and the ones we want to change between calls should be parameters that we pipe to the send to Helix template

Reply:
Probably I'm not following due to my lack of knowledge in Yaml but the intention is that `azure-pipelines` only reference this template once. No parameters/variables are specified in `azure-pipelines` for this. Things that are needed for Helix validation are defined in this file. 

From @chcosta's comment I got that I could have "constant" values as `parameters` and the rest as `variables` and then do something like:

```yaml
- template: /eng/common/templates/steps/send-to-helix.yml
      parameters:
        ${{parameters}}
        Variable1: $(_Variable1)
        Variable2: $(_Variable2)
        ...
```

Does that make sense?

Reply:
You add parameters if you want to call the template more than once with different values provided for certain fields.

If it's going to be called just once, then making them parameters won't be that useful.

I think we'll want an intermediate template at some point that keeps the values that won't change as constants, and will take the parameters that need to change and pass those to the helix template.

Reply:
I think that, conceptually, this feels a bit off to me.  To me, a template defines a generic set of steps that the caller configures.  In my mind, it's not just some function or encapsulation of functionality.  Generally, using templates as a black-box tends to make me feel yucky because I want to go to a single place to configure my build, not go to a root file and then follow that to another template, and then find another template so that I can align the functionality there with something that happened in some other level of the composed yaml.  

My general preference would be to make this a template that accepts parameters with reasonable defaults, and if you need to call it more than once and specify some alternative "_BuildConfig", then that makes sense.

Reply:
Changes made. Test build here: https://dnceng.visualstudio.com/internal/_build/results?buildId=58564